sdk/documenttranslation/Azure.AI.DocumentTranslation/tests/TranslationOperationTests.cs

```diff
@@ -0,0 +1,99 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Text;
+using System.Threading.Tasks;
+using Azure.Storage.Blobs;
+using Azure.Storage.Blobs.Models;
+using Azure.Storage.Sas;
+using NUnit.Framework;
+
+namespace Azure.AI.DocumentTranslation.Tests
+{
+    public class TranslationOperationTests : DocumentTranslationLiveTestBase
+    {
+        public TranslationOperationTests(bool isAsync) : base(isAsync) { }
+
+        private string StorageConnectionString = Environment.GetEnvironmentVariable("STORAGE_ACCOUNT_CONNECTION_STRING");
+
+        private List<string> Documents = new List<string>
+        {
+            "This is the first english test document",
+            "This is the second english test document"
+        };
+
+        private string GetSourceContainerName() => (TestContext.CurrentContext.Test.Name + IsAsync + "source").ToLower();
```

Comment:
I worry that the name of the container could be too long and might fail.
What if the container name is only `source + guid` ?
you could use the [GenerateAlphaNumericId](https://github.com/Azure/azure-sdk-for-net/blob/master/sdk/core/Azure.Core.TestFramework/src/TestRecording.cs#L229) function to guarantee that it will be unique and that it will work with out recording framework

Reply:
@maririos I didn't use a GUID because I didn't want to save the name as variable so that it's unique to each test within a class. This way it's unique but not saved as a variable in the class.

Reply:
if two pipelines are running at the same time, this will be a problem as the name of the container will be the same.

Other solution could be not to have the creation of the containers under `[SetUp]` but under a function that will return the SAS of each created container. Every test that needs a container will call that function. (like our current `GetClient()`).

We can also have a `[OneTimeTearDown]` method that will [list all containers](https://docs.microsoft.com/en-us/azure/storage/blobs/storage-blob-containers-list?tabs=dotnet#example-list-containers) in a storage account and then delete them, or, just let the CI delete whatever the CI created. (This last one is the approach that Python is taking)

Reply:
@maririos Sounds good. So the the creation function is called inside the test and then just leave the deletion to the CI.

Reply:
Yeah, we can try this for now and see how it scales